snuba/query/logical.py

```diff
@@ -169,6 +169,38 @@ def transform_expression_list(
             )
         )
 
+    TVisited = TypeVar("TVisited")
```

Comment:
I don't think this is used?

Reply:
errrr, it was used till 30 minutes ago. Then I forced the visitor to be parametrized on expressions. Will remove in a minute.